tests/scripts/release/.gitignore

```diff
@@ -0,0 +1,5 @@
+*.md
```

Comment:
maybe this should be tools/release/ or release/?

Reply:
Let's leave it here for now and move it once https://discuss.tvm.apache.org/t/establishing-a-home-for-tvm-dev-tools/13212 is settled